Code/GraphMol/Conformer.h

```diff
@@ -115,10 +115,14 @@ class RDKIT_GRAPHMOL_EXPORT Conformer {
 
   //! Get the number of atoms
   inline unsigned int getNumAtoms() const { return rdcast<unsigned int>(d_positions.size()); }
-
+  inline bool hasZCoords() const {
```

Comment:
This is clearly useful, but I would rather not have it as a method (it clutters the API for the class, which is nice to avoid).
How about moving it to a function here in Conformer.h?